config/defaults_template.yaml

```diff
@@ -36,7 +36,7 @@ languages:
 
 - language: php7
   buildImage: "{{ .BuildImagePrefix }}php:{{ .Version }}"
```

Comment:
Is the build image really "php"?

Reply:
I think in the makefile we pushed them to php7 instead of php 
https://github.com/grpc/test-infra/blob/29ec42a3716e76ff3822aef64f0c356f16a6a787/Makefile#L224

Reply:
> I think in the makefile we pushed them to php7 instead of php
> https://github.com/grpc/test-infra/blob/29ec42a3716e76ff3822aef64f0c356f16a6a787/Makefile#L224

Right, let's fix that too then. You can check in the latest release images.

Reply:
I did not fully understand what else needed to be updated, I believe this is the only place left as php. The make targets are all updated to php7. Let me know if I missed something. :)

Reply:
Right, sorry I missed it, I thought I changed it.